app/Http/Controllers/BookingAPIController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Booking;
+use Illuminate\Http\Request;
+
+class BookingAPIController extends Controller
+{
+    public function index()
+    {
+        $currentAndFutureBookings = Booking::with('room:id,name')
+            ->where('end', '>', now())
+            ->get()?->makeHidden(['updated_at', 'room_id'])
+            ->map(function ($booking) {
+                $booking->room->makeHidden(['id']);
+                return $booking;
+            })
```

Comment:
I believe the ID actually needs to stay - the front-end does (I think) use that information. Even if it doesn't, the rooms do have both id and name in the docs, so you should stick with that

Cool use of map tho, never considered using it for stuff like this.